### PR TITLE
feat: Instruct sentry-cli to create/upload il2cpp mappings

### PR DESCRIPTION
**Pull request by Swatinem**

### CODE REVIEW

**Comment by Swatinem** [comment]
sentry-cli `2.2.0` was released over the weekend which has the required flag:
https://github.com/getsentry/sentry-cli/releases/tag/2.2.0

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,7 +17,7 @@
 - Bump Native SDK to v0.4.18 ([#810](https://github.com/getsentry/sentry-unity/pull/810) & [#824](https://github.com/getsentry/sentry-unity/pull/824))
   - [changelog](https://github.com/getsentry/sentry-native/blob/master/CHANGELOG.md#0418)
   - [diff](https://github.com/getsentry/sentry-native/compare/0.4.15-7-g9eecb1b...0.4.18)
-- Bump CLI to v2.2.0 ([#826](https://github.com/getsentry/sentry-unity/pull/826))
+- Bump Sentry CLI to 2.2.0 ([#826](https://github.com/getsentry/sentry-unity/pull/826))
```
Comment (vaind) [inline CHANGELOG.md:20]: @bitsandfoxes 

1. We don't say "sentry native SDK", "Sentry Java SDK" so why Sentry CLI?
2. Same about the `v` prefix, which is there intentionally here and in other changelog entries above

Comment (bitsandfoxes) [reply CHANGELOG.md:20]: You're right, I got swept up by the moment. I'll correct it.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -3,8 +3,8 @@ name: CI
 on:
   push:
     paths-ignore:
-      - '**.md'
-      - '**.txt'
+      - "**.md"
```
[the remark this answers is not in the document]
Comment (Swatinem) [reply .github/workflows/ci.yml:6]: I think that was just my auto-formatter ;-) I didn’t do that explicitly.

[the remark this answers is not in the document]
Comment (vaind) [reply .github/workflows/ci.yml:6]: any chance you can get that disabled in this repo and revert?

**Review by vaind — APPROVED** [review]

**Review by bitsandfoxes — APPROVED** [review]

**Pull request merged**